### PR TITLE
Add commit linting and commit cli

### PR DESCRIPTION
**Pull request by Amrmak**
also using eslint:recommended eslint plugin instead of airbnb-base

### CODE REVIEW

File: .lintstagedrc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
 	"*.{png,jpeg,jpg,gif,svg}": ["imagemin-lint-staged", "git add"],
-	"*.js": ["npm run lint:js"],
-	"*.css": ["npm run lint:css"],
+	"*.js": ["eslint"],
```
Comment (ahmadalfy) [inline .lintstagedrc]: Why did you change these?

Comment (Amrmak) [reply .lintstagedrc]: Because the scripts referenced in the `.js` and `.css` executes linting on the whole scripts and styles folders respectively.
```
"lint:js": "eslint src/scripts/**/*.js --color",
"lint:css": "stylelint src/styles/**/*.css --color --fix",
```
Which I believe is not the intended behavior, we only want to lint the staged files
